apps/sendsignal.go

```diff
@@ -0,0 +1,29 @@
+package apps
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+)
+
+func SendSignal(ctx context.Context, app string, signal string, args []string) error {
+	if len(args) == 0 {
+		return errgo.New("at least one container name should be given")
+	}
+	c, err := config.ScalingoClient(ctx)
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client to send signal to application containers")
+	}
+
+	for _, container := range args {
```

Comment:
It's not a `container` (which would suggest some kind of `Container` structure) you have here. You have a `containerName` I think.